CMakeLists.txt

```diff
@@ -35,9 +35,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 message(STATUS "pgmoneta ${VERSION_STRING}")
 
-set(generation TRUE)
-set(check TRUE)
-set(container FALSE)
+set(generation FALSE)
+set(check FALSE)
+set(container TRUE)
```

Comment:
Why did you change these ?

Reply:
It's just a basic structure, and it's convenient to test locally to avoid manually generating the manifest.